addons/repair/views/repair_views.xml

```diff
@@ -44,6 +45,7 @@
                <sheet string="Repairs order">
                     <div class="oe_button_box" name="button_box">
                         <field name="invoice_id" invisible="1"/>
+                        <button name="%(action_repair_move_lines)d" type="action" string="Product Moves" class="oe_stat_button" icon="fa-exchange" attrs="{'invisible': [('state', 'in', ('draft', 'confirmed', 'ready', 'under_repair'))]}"/>
```

Comment:
IMO, It should be more readable be inverse the condition:
`{'invisible': [('state', 'not in', ('done', 'cancel'))]}`